tests/network-tests/run-migration-tests.sh

```diff
@@ -68,33 +133,174 @@ function post_migration_hook() {
+function pre_migration_cli() {
+  sleep 10 # needed otherwise docker image won't be ready yet
   # Display runtime version
   yarn workspace api-scripts tsnode-strict src/status.ts | grep Runtime
 
-  echo "Performing migration tests"
+  # assume older version of joystream-cli is installed globally. So we run these commands to
+  # work against older runtime. Assert it is version  `@joystream/cli/0.5.1` ?
+  yarn joystream-cli --version
+
+  yarn joystream-cli account:choose --address 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY # Alice
+  echo "creating 1 channel"
+  yarn joystream-cli content:createChannel --input=./assets/TestChannel.json --context=Member || true
+  echo "adding 1 video to the above channel"
+  yarn joystream-cli content:createVideo -c 1 --input=./assets/TestVideo.json || true
+
+  # Confirm channel and video created successfully
+  yarn joystream-cli content:videos 1
+  yarn joystream-cli content:channel 1
+}
 
-  post_migration_hook
+#######################################
+# Verifies post migration assertions
+# Globals:
+#   None
+# Arguments:
+#   None
+#######################################
+function post_migration_cli() {
+  echo "*** verify existence of the 5 new groups ***"
+  yarn joystream-cli working-groups:overview --group=operationsAlpha
+  yarn joystream-cli working-groups:overview --group=operationsBeta
+  yarn joystream-cli working-groups:overview --group=operationsGamma
+  yarn joystream-cli working-groups:overview --group=curators
+  yarn joystream-cli working-groups:overview --group=distributors
+
+  echo "*** verify previously created channel and video are cleared ***"
+  # Allow a few blocks for migration to complete
+  sleep 12
+  
+  # FIXME: Howto assert these fail as expected. They should report video and channel do no exist
+  # Can we get json output to more easily parse result of query?
+  set +e
+  yarn joystream-cli content:channel 1
+  if [ $? -eq 0 ]; then
+    echo "Unexpected channel was found"
+    exit -1
+  fi
+  # This cammand doesn't give error exit code if videos not found in a channel
+  yarn joystream-cli content:videos 1
+}
+
+# entrypoint
+function main {
+    CONTAINER_ID=""
+
+    echo "**** CREATING EMPTY CHAINSPEC ****"
+    create_initial_config
+    create_chainspec_file
+    convert_chainspec
+    echo "**** EMPTY CHAINSPEC CREATED SUCCESSFULLY ****"
+
+    # use forkoff to update chainspec with the live state + update runtime code
+    fork_off_init
+
+    echo "***** STARTING NODE WITH FORKED STATE *****"
+    JOYSTREAM_NODE_TAG=$TARGET_RUNTIME_TAG
+    CONTAINER_ID=$(start_node)
+
+    # verify that the number of outstanding channels & videos == 0
+    # if ( $POST_MIGRATION_CLI_ASSERTIONS ); then
+    # 	# verify assertion using cli
+    # 	echo "***** POST MIGRATION CLI *****"
+    # 	post_migration_cli
+    # fi
+    
+    if ( $POST_MIGRATION_ASYNC_ASSERTIONS ); then
+	# verify assertion using typsecript
+	sleep 10 # TODO: 3m and 52 sec needed!!
```

Comment:
It took a little bit less on my machine, but yes its not surprising since the chainspec file is around 100MB.